### DISCUSSION TITLE
Is it possible to parallelize fooocus?

### DISCUSSION THREAD

**Discussion by quiet-observer** [lllyasviel/Fooocus#1981] [General]
Right now the queries get queued one after another. Meaning if two, three, hundred computers connect to the model they will only generate one after another. Do you think it's possible to generate multiple images at the same time?

**Comment by poor7** [comment]
There will be no acceleration. What is the point?

**Reply by DoWhileGeek** [reply]
@poor7 if the other computers have graphics cards, there is definitely acceleration to be had. Who doesnt have a hobby render farm these days? 🥁

**Comment by DoWhileGeek** [comment]
I _may_ be interested in contributing this feature.

**Reply by quiet-observer** [reply]
@DoWhileGeek @poor7 Ya what I mean is. Have you tried to run fooocus in the cloud? If you try to run two session, and enter the prompts simultaneously. It'll only generate the images, for one session at a time. Do you think there is a way to generate two session at the same time? Running on one system, with one gpu.
![fooocus parrallel](https://github.com/lllyasviel/Fooocus/assets/156555575/08be0051-719d-4805-a9f7-c33035111762)

**Reply by DoWhileGeek** [reply]
Oh no, I only run locally. I see littoe to no use in two sessions sharing a gpu, when one session completely saturates it.

**Reply by quiet-observer** [reply]
Have you tried running multiple sessions, at the same time?
